packages/ui/src/containers/ApplyByRoleContainer/ApplyByRoleContainer.test.tsx

```diff
@@ -1,12 +1,71 @@
-import { render } from "../../../utils/jest-apollo";
+import { getProject } from "@eden/package-mock";
+import userEvent from "@testing-library/user-event";
+import { RouterContext } from "next/dist/shared/lib/router-context";
+
+import { render, screen } from "../../../utils/jest-apollo";
+import { createMockRouter } from "../../../utils/test-utils/createMockRouter";
 import { ApplyByRoleContainer } from "./";
 
-describe("ApplyByRoleContainer", () => {
-  it("renders without throwing", () => {
-    const { container } = render(
-      <ApplyByRoleContainer onViewProject={(val) => console.log(val)} />
-    );
+jest.mock("react-confetti");
+
+test("User can get to the YOU DID IT modal ", async () => {
+  const user = userEvent.setup();
+
+  await render(
+    <RouterContext.Provider value={createMockRouter({})}>
+      <ApplyByRoleContainer
+        onViewProject={(val) => console.log(val)}
+        loadingProject={false}
+        project={getProject()}
+      />
+    </RouterContext.Provider>
```

Comment:
With `MockRouter`:

```suggestion
    <MockRouter>
      <ApplyByRoleContainer
        onViewProject={(val) => console.log(val)}
        loadingProject={false}
        project={getProject()}
      />
    </MockRouter>
```

Instead of needing to know about `RouterContext` _and_ `createMockRouter` in order to mock, devs just need to know about `MockRouter`.